### PR TITLE
Update to accomodate global db

### PR DESCRIPTION
**Pull request by mrrobot47**
Depends on https://github.com/EasyEngine/site-command/pull/128

### CODE REVIEW

File: src/WordPress.php [hunk 1]
```diff
@@ -406,9 +414,15 @@ private function maybe_verify_remote_db_connection() {
...
+		if ( GLOBAL_DB !== $this->site_data['db_host'] ) {
+			\EE::log( 'Verifying connection to remote database' );
+		}
+		$img_versions = \EE\Utils\get_image_versions();
+		if ( GLOBAL_DB === $this->site_data['db_host'] ) {
+			$network = ( GLOBAL_DB === $this->site_data['db_host'] ) ? "--network='" . GLOBAL_NETWORK . "'" : '';
+		}
 
-		if ( ! \EE::exec( sprintf( "docker run -it --rm --network='%s' mysql sh -c \"mysql --host='%s' --port='%s' --user='%s' --password='%s' --execute='EXIT'\"", $this->site_data['site_url'], $this->site_data['db_host'], $this->site_data['db_port'], $this->site_data['db_user'], $this->site_data['db_password'] ) ) ) {
+		if ( ! \EE::exec( sprintf( "docker run -it --rm %s easyengine/mariadb:%s sh -c \"mysql --host='%s' --port='%s' --user='%s' --password='%s' --execute='EXIT'\"", $network, $img_versions['easyengine/mariadb'], $this->site_data['db_host'], $this->site_data['db_port'], $this->site_data['db_user'], $this->site_data['db_password'] ) ) ) {
```
Comment (kirtangajjar) [inline src/WordPress.php]: Here `$network` variable will not be initialised if `GLOBAL_DB !== $this->site_data['db_host']`

File: src/WordPress.php [hunk 2]
```diff
@@ -529,12 +542,12 @@ private function wp_download_and_config( $assoc_args ) {
...
+				$name            = $this->site_data['db_name'];
+				$check_db_exists = sprintf( "docker-compose exec php bash -c \"mysqlshow --user='%s' --password='%s' --host='%s' --port='%s' '%s'\"", $this->site_data['db_user'], $this->site_data['db_password'], $this->site_data['db_host'], $this->site_data['db_port'], $name );
 
 				if ( ! \EE::exec( $check_db_exists ) ) {
 					\EE::log( sprintf( 'Database `%s` does not exist. Attempting to create it.', $this->site_data['db_name'] ) );
-					$create_db_command = sprintf( 'docker-compose exec php bash -c "mysql --host=%s --port=%s --user=%s --password=%s --execute="CREATE DATABASE %s;"', $this->site_data['db_host'], $this->site_data['db_port'], $this->site_data['db_user'], $this->site_data['db_password'], $this->site_data['db_name'] );
+					$create_db_command = sprintf( 'docker-compose exec php bash -c "mysql --host=%s --port=%s --user=%s --password=%s --execute="CREATE DATABASE %s;""', $this->site_data['db_host'], $this->site_data['db_port'], $this->site_data['db_user'], $this->site_data['db_password'], $this->site_data['db_name'] );
```
Comment (kirtangajjar) [inline src/WordPress.php]: Won't the double quote need escaping here :thinking:

Comment (mrrobot47) [reply src/WordPress.php]: No

File: src/WordPress.php (hunk 1 above)
Comment (mrrobot47) [reply src/WordPress.php]: Yes. In that case network is not needed, empty string is needed and that is the case.

**Pull request merged**